carbon-design-system/ibm-security
# fix(type-layout): flatten styles, update padding, and do not clone cells

## Affected issues

- Resolves #399 

## Proposed changes

- flatten styles
- fix specificity of styles + update padding to specifically apply to each size
- stop clone cell elements in component

## Testing instructions

- I suggest pulling up this storybook branch deploy next to the `master` branch storybook deploy. Have them both size-by-size, and then check each variation. 🕵️‍♀️  They should be exactly the same.
